cli/tools/fmt.rs

```diff
@@ -84,23 +91,37 @@ fn format_markdown(
           "typescript" => "ts",
           rest => rest,
         };
-        let fake_filename =
-          PathBuf::from(format!("deno_fmt_stdin.{}", extension));
 
-        let mut codeblock_config = ts_config.clone();
-        codeblock_config.line_width = line_width;
-        dprint_plugin_typescript::format_text(
-          &fake_filename,
-          &text,
-          &codeblock_config,
-        )
+        if matches!(extension, "json" | "jsonc") {
+          let mut json_config = get_json_config();
+          json_config.line_width = line_width;
+          dprint_plugin_json::format_text(&text, &json_config)
```

Comment:
I don't think this bit is exercised in tests, could you add some codeblock with JSON to ensure it's tested?